commit 0e3811d7914f209858cf4e445221fd65644cc94b
Author: Andreas Schwab <schwab@suse.de>
Date:   Thu Mar 12 18:07:00 2015 +0100

    aarch64: fix ioctl decoding
    
    * linux/aarch64/ioctls_inc0.h: Rename from ioctls_inc1.h.
    * linux/aarch64/ioctls_inc1.h: Rename from ioctls_inc0.h.
    * linux/aarch64/ioctls_arch0.h: Rename from ioctls_arch1.h.
    * linux/aarch64/ioctls_arch1.h: Rename from ioctls_arch0.h.

---
 linux/aarch64/ioctls_arch0.h |    2 +-
 linux/aarch64/ioctls_arch1.h |    2 +-
 linux/aarch64/ioctls_inc0.h  |    2 +-
 linux/aarch64/ioctls_inc1.h  |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

--- a/linux/aarch64/ioctls_arch0.h
+++ b/linux/aarch64/ioctls_arch0.h
@@ -1 +1 @@
-/* Generated by ioctls_gen.sh from definitions found in $linux/arch/arm64/include/ tree. */
+#include "arm/ioctls_arch0.h"
--- a/linux/aarch64/ioctls_arch1.h
+++ b/linux/aarch64/ioctls_arch1.h
@@ -1 +1 @@
-#include "arm/ioctls_arch0.h"
+/* Generated by ioctls_gen.sh from definitions found in $linux/arch/arm64/include/ tree. */
--- a/linux/aarch64/ioctls_inc0.h
+++ b/linux/aarch64/ioctls_inc0.h
@@ -1 +1 @@
-#include "64/ioctls_inc.h"
+#include "arm/ioctls_inc0.h"
--- a/linux/aarch64/ioctls_inc1.h
+++ b/linux/aarch64/ioctls_inc1.h
@@ -1 +1 @@
-#include "arm/ioctls_inc0.h"
+#include "64/ioctls_inc.h"
